test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -2814,6 +2814,29 @@ def forward(self, x):
         x = torch.randn(2, 3, 4)
         self.run_test(Rand(), x)
 
+    def test_random_dtype(self):
```

Comment:
Could you break this into two test cases?

Additionally, 
I would include the expected behavior as part of the test method name. To make what a test is testing clear: `test_<method or function name>_<expected behavior>_[when_<condition>]`.

e.g. `test_randn_maintains_the_correct_dtype` etc

Reply:
For the naming, I feel the current approach is sufficient. The purpose of this file is to test the correctness (consistency) between PyTorch's outputs and ONNX runtime's outputs, so adding `maintains_the_correct` seems superfluous.

Ok with breaking into two test cases.

Reply:
ok